app/decorators/report_decorator.rb

```diff
@@ -16,4 +16,12 @@ def total_learning_time
       "#{hour}時間#{minute}分"
     end
   end
+
+  def number
+    if serial_number == 1
+      "初日報"
+    else
+      "#{serial_number}"
```

Comment:
```suggestion
      serial_number.to_i
```

ですかね。式展開不要なときは`#{}`は外したいですね

Reply:
レビューありがとうございます。
上記修正致しました🙇‍♂️